CONTRIBUTING.md

```diff
@@ -27,3 +27,17 @@ Submitting a pull request
   * Be sure to select the appropriate labels/milestones for your pull request.
   * If your pull request applies to an issue, be sure to mention it in the description.
 * The new pull request will be broadcasted in #gazelle, and the Core team will review it and leave feedback.
+
+Privacy Considerations
+----------------------
+
+You might not want to have your real name linked with this project. Here we provide some hints that should help contributing under a pseudonym.
+
+* If you want to participate in the discussion, create pull requests, or edit the wiki, consider creating a separate GitHub user account. We cannot delete any pull requests or issues made in the bug tracker.
+* If you're using git for other projects as well, check if commits you make on this project use your real name. GitHub uses the email address in commit to associate them to your account. The relevant configuration options are listed below. The used email address does not need to be real. If you need to fix commits you've already made, see `git commit --amend --reset-author`.
+```
+git config --local user.name
+git config --local user.email
+git config --local user.signingkey
+```
+* Consider using a VPN or [Tor](https://torproject.org) for additional anonimity. Don't forget to proxy any `git` operations as well.
```

Comment:
I would also add smtg like this

In order to do so, one solution in linux is to:
- install tor and start corresponding service
- install socat
- add the following content to '~/ssh/ssh_config' or better (since it would also work if you push in root) to '/etc/ssh/ssh_config'
```
Host github
  HostName github.com
  IdentityFile /path/to/your/file
  User git
  ProxyCommand socat STDIO SOCKS4A:127.0.0.1:%h:%p,socksport=9050```

An other (and probably the best one) is to work inside a dedicaced tor vm (like https://www.whonix.org/)